policyengine_us/variables/gov/states/wv/tax/income/credits/sctc/wv_taxable_property_value.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class wv_taxable_property_value(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia taxable property value"
+    defined_for = StateCode.WV
+    unit = USD
+    definition_period = YEAR
+    reference = "https://code.wvlegislature.gov/11-21-21/"
+
```

Comment:
Would this work instead of the formula?
```suggestion
    adds = "assessed_property_value"
    subtracts = "wv_homestead_exemption"
```

Reply:
Yes